core/include/mutex.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
- *               2013, 2014 Freie Universität Berlin
+ *               2013 - 2017 Freie Universität Berlin
```

Comment:
please keep spacing consistent in the license header (I prefer without spaces and it seems to be the goto in the code base)